### PR TITLE
Fix #404, to support multiple labels in different languages

### LINKED ISSUE #404: Labels should be localized
The human-friendly labels do not indicate the language used and the classes do not permit providing labels in multiple languages. Consider adding a loop with language + label_value.

### PR DESCRIPTION
**Pull request by sunghee-hwang**
ISO 639-1 is more specific and direct reference than BCP47.

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/429.html" title="Last updated on May 26, 2023, 8:19 PM UTC (2a2cc3c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/429/795601b...2a2cc3c.html" title="Last updated on May 26, 2023, 8:19 PM UTC (2a2cc3c)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -968,14 +966,22 @@ A mix presentation may contain one or more sub-mixes. Common use-cases may speci
 ```
 class mix_presentation_obu() {
   leb128() mix_presentation_id;
-  mix_presentation_annotations();
+  leb128() count_label;
+  for (i = 0; i < count_label; i++) {
+    unsigned int (16) language_label;
```
Comment (cconcolato) [inline index.bs:971]: why is the type `unsigned int (16)` and not simply `string`?

File: index.bs [hunk 2]
```diff
@@ -968,14 +966,22 @@ A mix presentation may contain one or more sub-mixes. Common use-cases may speci
...
+  }
+  for (i = 0; i < count_label; i++) {
+    mix_presentation_annotations();
```
Comment (cconcolato) [inline index.bs:974]: why do you need a class and not simply use `string`

**Review by cconcolato — CHANGES_REQUESTED** [review]

File: index.bs [hunk 3]
```diff
@@ -197,11 +197,11 @@ url: https://www.iso.org/standard/77752.html#; spec: MP4-PCM; type: property;
...
-	"BCP47": {
-		"title": "BCP 47",
-		"status": "Best Practice",
-		"publisher": "IETF",
-		"href": "https://www.rfc-editor.org/info/bcp47"
+	"ISO-639-1": {
+		"title": "ISO 639-1",
+		"status": "Standard",
+		"publisher": "ISO",
+		"href": "https://www.iso.org/standard/22109.html"
```
Comment (cconcolato) [inline index.bs:204]: ISO 639-1 is restricted compared to BCP-47. AFAIK, modern standards refer to BCP-47.

File: index.bs (hunk 1 above)
Comment (sunghee-hwang) [reply index.bs:971]: Intention was to represent Alpha-2 code.

File: index.bs (hunk 2 above)
Comment (sunghee-hwang) [reply index.bs:974]: Annotation may be updated for a future version. It was proponent's proposal.

File: index.bs (hunk 1 above)
Comment (sunghee-hwang) [reply index.bs:971]: I made further update to clarify 2CC language_label.

**Comment by sunghee-hwang** [comment]
@cconcolato, please review it again.

File: index.bs [hunk 4]
```diff
@@ -279,10 +279,8 @@ All of syntax elements conform to [=Syntactic Description Language=] specified i
 
  <b>string</b> indicates the type of a string which is terminated by null of one byte (i.e. 0x00).
```
Comment (cconcolato) [inline index.bs:280]:
```suggestion
<b>string</b> indicates a null-terminated (i.e. ending at the first byte set to 0x00), UTF-8 encoded as defined in [[!RFC3629]] and whose length shall be limited to 128 bytes. 
```

File: index.bs [hunk 5]
```diff
@@ -279,10 +279,8 @@ All of syntax elements conform to [=Syntactic Description Language=] specified i
...
  
- <b>syntaxName</b> is a human readable label whose byte representation consists of <b>two-letter primary language subtags</b> and <b>two-letter region subtags</b> which are connected by hyphen("-"), and followed by bytes representation of [=UTF-8_Enc(label)=]. Its length in bytes is limited to 128.
+ <b>syntaxName</b> is a human readable label whose byte representation consists of [=UTF-8_Enc(label)=]. Its length in bytes is limited to 128.
```
Comment (cconcolato) [inline index.bs:282]:
```suggestion
<b>syntaxName</b> is a human readable label.
```

File: index.bs
Comment (cconcolato) [inline index.bs:1]: I also suggest deleting the section 
```
### Function UTF-8 Encoding ### {#convention-function-utf8}
```

**Review by cconcolato — CHANGES_REQUESTED** [review]

File: index.bs
Comment (sunghee-hwang) [reply index.bs:1]: @cconcolato , deleted the section. Please review it again. Thanks in advance.

**Review by cconcolato — APPROVED** [review]

**Comment by sunghee-hwang** [comment]
As agreed, I do merge it.

**Pull request merged**